internal/anelace/argparse.go

```diff
@@ -120,6 +120,13 @@ func NewFromArgv(argv []string) (anl *Anelace) {
 				emRootsJsonl:  nil,
 				emCarV1Stream: nil,
 			},
+
+			// some opinionated defaults
+			requestedNodeEncoder: "unixfsv1_merkledag-compat-protobuf",
```

Comment:
Throw away `compat-protobuf`, it serves little purpose aside from confusing parsers ( in that mode `2` frames come before `1` frames )